src/lib/pages/frontsite/Literature/LiteratureSearch/SearchGuideLink/SearchGuideLink.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+export default class SearchGuideLink extends Component {
+  render() {
+    return (
+      <div className="ui left aligned container">
```

Comment:
Why not being more "React"ive :)
```suggestion
<Container textAlign='left'>...</Container>
```
See React-SemanticUI doc instead of SemanticUI: https://react.semantic-ui.com/elements/container/#variations-alignment

Reply:
Definitely.